go/src/meguca/templates/body.go

```diff
@@ -527,6 +529,11 @@ func (c *bodyContext) parseCommands(bit string) {
 	case "8ball":
 		inner = append(inner, val.Eightball...)
 		c.state.iDice++
+	case "pyu", "pcount":
+		if (c.state.pyu) {
```

Comment:
This will cause an entire row of bugs in command parsing, if pyu is toggled on and off. Perhaps strip `#pyu` in `parser/body.go`?